web/src/app/schedules/temp-sched/TempSchedAddShiftForm.tsx

```diff
@@ -85,7 +85,8 @@ export default function TempSchedAddShiftForm({
                 .plus({ hours: parseFloat(nextVal) })
                 .toISO()
             }}
-            min={0.25}
+            step={1}
```

Comment:
Looks like having the step set to 1 is preventing the form from submitting if the value isn't a whole number, causing some of the cypress tests to be flaky.